### DISCUSSION TITLE
Add a toggle to despawn a nob when the owner disconnects

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#776] [Feature Request]
It would be convenient to have a toggle on NetworkObjects to remove ownership (prevent despawning) when its owner disconnects.
A global toggle could also be added to one of the manager components on the NetworkManager object that would affect all objects.

**Comment by FirstGearGames** [comment]
Any thoughts on what to name this field?
It would appear nested if the default despawn type is destroy 
![image](https://github.com/user-attachments/assets/827d1da3-39a9-48d5-a119-513dde944e6a)

**Comment by maxkratt** [comment]
I suppose the setting wouldn't work on the pool option then?
In which case a good name would make sense to be specific to destroying instead of ownership removal, what do you think?

**Comment by FirstGearGames** [comment]
Yea that's what I figured but drawing a blank

**Comment by maxkratt** [comment]
Destroy On Owner Disconnect is the main thing that comes to mind

**Comment by FirstGearGames** [comment]
Added to roadmap.
